CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * [ENHANCEMENT] Compactor: Exposing Thanos accept-malformed-index to Cortex compactor. #5334
 * [ENHANCEMENT] Update Go version to 1.20.4. #5299
 * [ENHANCEMENT] Log: Avoid expensive log.Valuer evaluation for disallowed levels. #5297
+* [ENHANCEMENT] Improving Performance on the API Gzip Handler. #5347
```

Comment:
Do we need the changelog? It should be an internal details and users don't need to know about that.

Reply:
hummm... i think you are right... Should i remove in another PR now?

Reply:
Yes you can update it in another pr